repo_guidelines.md

```diff
@@ -1,9 +1,10 @@
-# Project Guidelines
+# Project Repo Guidelines & Template
 
 Each project repository **requires**, at a minimum:
 
-1. Contributing Guidelines
-1. License
+1. [Contributing Guidelines](#contributing-guidelines)
+1. [License](#license)
+1. [Readme](#readme) including several elements listed below
```

Comment:
So I just discovered that this link doesn’t work because GitHub wraps the whole rendered markdown file in a div with the ID `readme` :(

Not sure if we should re-title the section or just add a hidden anchor element with a custom ID for linking.

Reply:
Interesting, kinda makes sense. I re-titled each of those sections e.g. "about the readme" because on reflection that's a bit clearer anyway